configure.py

```diff
@@ -1766,6 +1774,29 @@ def configure_inspector(o):
                        options.without_ssl)
   o['variables']['v8_enable_inspector'] = 0 if disable_inspector else 1
 
+def configure_section_file(o):
+  try:
+    proc = subprocess.Popen(['ld.gold'] + ['-v'], stdin = subprocess.PIPE,
+                            stdout = subprocess.PIPE, stderr = subprocess.PIPE)
+  except OSError:
+    return 0
```

Comment:
Is failing silently what we want?  Do we want to raise a warning that the user has requested that we use `ld.gold` but it can not be found?